possible panic in wait/host_port.go

In line 73 of file wait/host_port.go in function WaitUntilReady the connvariable is called before checking for err. So if conn is nil it will not continue normally.
Suggestion is to move the line to be after the err check.

`go version go1.13.5 windows/amd64`

I got that panic today:
```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0xac8f6b]

goroutine 7 [running]:
testing.tRunner.func1(0xc0001f7500)
	C:/Go/src/testing/testing.go:874 +0x616
panic(0xe627c0, 0xd99890)
	C:/Go/src/runtime/panic.go:679 +0x1ee
github.com/testcontainers/testcontainers-go/wait.(*HostPortStrategy).WaitUntilReady(0xc0002d5d20, 0x1143140, 0xc0002e83c0, 0x11432c0, 0xc00035c190, 0x0, 0x0)
	D:/Projects/Go/pkg/mod/github.com/testcontainers/testcontainers-go@v0.1.0/wait/host_port.go:73 +0x4fb
github.com/testcontainers/testcontainers-go.(*DockerContainer).Start(0xc00035c190, 0x1143100, 0xc000018158, 0x0, 0x0)
	D:/Projects/Go/pkg/mod/github.com/testcontainers/testcontainers-go@v0.1.0/docker.go:142 +0x1ad
github.com/testcontainers/testcontainers-go.GenericContainer(0x1143100, 0xc000018158, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xf2f760, 0x14, ...)
	D:/Projects/Go/pkg/mod/github.com/testcontainers/testcontainers-go@v0.1.0/generic.go:29 +0x2fb
```

--- a/wait/host_port.go
+++ b/wait/host_port.go
@@ -3,12 +3,13 @@ package wait
 import (
 	"context"
 	"fmt"
-	"github.com/pkg/errors"
 	"net"
 	"os"
 	"strconv"
 	"syscall"
 	"time"
+
+	"github.com/pkg/errors"
 
 	"github.com/docker/go-connections/nat"
 )
@@ -70,6 +71,9 @@ func (hp *HostPortStrategy) WaitUntilReady(ctx context.Context, target StrategyT
 	address := net.JoinHostPort(ipAddress, portString)
 	for {
 		conn, err := dialer.DialContext(ctx, proto, address)
+		if err != nil {
+			return err
+		}
 		defer conn.Close()
 		if err != nil {
 			if v, ok := err.(*net.OpError); ok {

